providers/openebs/installers/operator/0.6/litmusbook/openebs_setup.yaml

```diff
@@ -0,0 +1,54 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: litmus-openebs-setup
+  namespace: litmus 
+spec:
+  template:
+    metadata:
+      name: litmus
+    spec:
+      serviceAccountName: litmus
+      restartPolicy: Never
+      containers:
+      - name: ansibletest
+        image: openebs/ansible-runner:ci
+        imagePullPolicy: Always
+        env: 
+          - name: mountPath
+            value: /mnt/openebs
+          - name: ANSIBLE_STDOUT_CALLBACK
+            value: actionable
+          - name: MAYA_APISERVER_IMAGE
+            value:
+          - name: OPENEBS_PROVISIONER_IMAGE
+            value:
+          - name: OPENEBS_SNAPSHOT_CONTROLLER_IMAGE
+            value:
+          - name: OPENEBS_SNAPSHOT_PROVISIONER_IMAGE
+            value:
+          - name: OPENEBS_IO_JIVA_CONTROLLER_IMAGE
+            value:
+          - name: OPENEBS_IO_JIVA_REPLICA_IMAGE
+            value:
+          - name: OPENEBS_IO_VOLUME_MONITOR_IMAGE
+            value:
+          - name: OPENEBS_IO_JIVA_REPLICA_COUNT
+            value:
```

Comment:
While I understand that it is expected of CI systems to fill in the  ENV values, wanted to confirm if the job can be run as-is (so defaults from the operator obtained from the repo take effect)

Reply:
ya if env values are not empty it will take the values present in the openebs operator .